lib/parallel_tests/cli.rb

```diff
@@ -87,6 +87,17 @@ def report_results(test_results)
       results = @runner.find_results(test_results.map { |result| result[:stdout] }*"")
       puts ""
       puts @runner.summarize_results(results)
+      report_failures(test_results) if any_test_failed?(test_results)
```

Comment:
This is here to help us debug randomly failing tests. We'd like to add some tests for this as well.